tests/io/test_data_access_map_file.py

```diff
@@ -11,12 +11,21 @@
 from dfastmi.io.map_file import MapFile
 
 
-@pytest.fixture
-def map_file() -> MapFile:
+def open_map_file() -> MapFile:
     filename = "tests/files/e02_f001_c011_simplechannel_map.nc"
     return MapFile(filename)
 
 
+def open_fou_file() -> MapFile:
```

Comment:
again, would not do this, MapFile != FouFile. Please implement something like

            (abstract) 
           OutputFile
                   ^
                    | 
_______________________________
|                                             |
MapFile                       FouFile